src/main/java/com/unzer/payment/service/PaymentService.java

```diff
@@ -190,6 +192,7 @@ private ApiIdObject getJsonObjectFromTypeId(String typeId) {
             case PF_EFINANCE:
             case UNZER_PAYLATER_INVOICE:
             case KLARNA:
+            case CLICK_TO_PAY:
                 return new ApiIdObject();
```

Comment:
Should we have a custom object here ?

Reply:
Are there any specific properties retuned in googlepay creation response, that dont exist in other types? If not, the generic object should be fine here.